docs/product-assembly_pam-on-openshift/src/main/asciidoc/main.adoc

```diff
@@ -0,0 +1,30 @@
+[id='assembly_dm-7.0-release-notes']
+
+include::product-shared-docs/document-attributes.adoc[]
+
+= Deploying {PRODUCT} {PRODUCT_VERSION} on OpenShift
+// Context attribute is assembly specific and enables module reuse between assemblies.
+:imagesdir: topics/product-shared-docs/images
+:context: dm-7.0-release-notes
+include::product-shared-docs/author-group.adoc[]
+
+// Purpose statement for the assembly
+These release notes highlight new features in {PRODUCT} {PRODUCT_VERSION}, list features that are in technical preview, and include any known issues.
```

Comment:
Is there actually something in tech preview, or is this just standard language?

Reply:
Boilerplate.